src/locales/es/modifier-select-ui-handler.ts

```diff
@@ -0,0 +1,14 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const modifierSelectUiHandler: SimpleTranslationEntries = {
+  "transfer": "Transferir",
+  "reroll": "Actualizar",
+  "lockRarities": "Bloquear Rarezas",
+  "checkTeam": "Ver Equipo",
+  "transferDesc": "Transfiere un objeto que lleva un Pokémon a otro.",
+  "rerollDesc": "Gasta dinero para actualizar las opciones de objetos.",
+  "lockRaritiesDesc": "Bloquea las rarezas de los objetos al actualizar (afecta el costo de actualización).",
+  "checkTeamDesc": "Revisa tu equipo o usa un objeto que cambia de forma.",
+  "rerollCost": "{{formattedMoney}} ₽",
+  "itemCost": "{{formattedMoney}} ₽"

```

Comment:
Small capitalisation change for consistency with the other occurence of the same word. French did the same.

```suggestion
  "transfer": "Transferir",
  "reroll": "Actualizar",
  "lockRarities": "Bloquear rarezas",
  "checkTeam": "Ver Equipo",
  "transferDesc": "Transfiere un objeto que lleva un Pokémon a otro.",
  "rerollDesc": "Gasta dinero para actualizar las opciones de objetos.",
  "lockRaritiesDesc": "Bloquea las rarezas de los objetos al actualizar (afecta el costo de actualización).",
  "checkTeamDesc": "Revisa tu equipo o usa un objeto que cambia de forma.",
  "rerollCost": "{{formattedMoney}} ₽",
  "itemCost": "{{formattedMoney}} ₽"

```